raiden-ts/src/transfers/epics/mediate.ts

```diff
@@ -20,6 +25,46 @@ function shouldMediate(action: transferSigned, address: Address, { caps }: Raide
   return !!isMediationEnabled && isntTarget;
 }
 
+function filterRoutes(
+  routes: readonly RouteMetadata[],
+  state: RaidenState,
+  tokenNetwork: Address,
+  { address, log }: Pick<RaidenEpicDeps, 'address' | 'log'>,
+) {
+  const allOpenPartners = new Set(
+    Object.values(state.channels)
+      .filter(
+        (channel) => channel.state === ChannelState.open && channel.tokenNetwork === tokenNetwork,
```

Comment:
This is slightly inefficient, since I've to iterate twice, but as the channel's count is expected to be low, in name of readability, I've implemented it.